### DISCUSSION TITLE
Plotting in different networks with differen Wallets

### DISCUSSION THREAD

**Discussion by LachenderTeufel** [Chia-Network/chia-blockchain#4403] [Q&A]
Hi Guys,
I guess I made a little Int!?

I started with chia already a few weeks ago. Just plotting and farming with my Gaming PC.
After thinking about investing something in the chia project I came up with the Idea to plot faster my free Space (20TB).
So I started plotting on my gaming Notebook/second PC/Working PC and so one.
I although accomplished to convince our IT guy in the company to use the desktop PC for plotting over night.

Of course I didnt read the following instruction: https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines
The IT guy make it even worse and created on every PC a new wallet and started plotting.
Now my questions:
1. Can I make now the above instructions to my network? Or is it to late and I need to plott it all again?
2. Do I need to set up the same on the company plotter?
3. Can I use the plots from the company pc with different wallets?

I´m doing all of this with the GUI in Windows.

Hope someone has an Idea how to fix the mess.

Thanks!

**Comment by SackofFluor** [comment]
No, the plots only work with your wallets, so they have to be reported (AFAIK)
If you enter the 24-keyword phrase to every computer at the company, everybody who gets wind of this action could use the CLI or GUI from those machines to steal your coins.

Thats why there are the instructions you linked above where mere "farmers" don't have all the major keys.

Best

Sacko

EDIT:
Wenn es nicht deine Firma ist, ist sowas definitiv in gefährliches Spiel und kann mit einer fristlosen Kündigung quittiert werden!
Zumal du ja auch jemand zusätzlich eingeweiht hast :/

**Reply by LachenderTeufel** [reply]
Hi, thats what I expected... So I need delete the plots I was plotting in the company.
The other ones at home are all made with my wallet adress.

Now the question, how do I identify the "wrong" plots?
About stealing I´m not worried, Due to the fact that these actions are running when everyone is leaving and they have no possibility to reenter the building or something like this.

Bzgl. der Kündigung, Chef ist eingeweiht und eine Schriftliche Genehmigung erteilt. Einzige Bedingung ist bei dem ersten Chia gibt es ein grillen in der Mittagspause ;)

**Reply by SackofFluor** [reply]
Ok, Super! Cool wenn man solche Chefs hat!

So.. to identify "wrong" plots...

In the CLI you can use "chia plots verify" and I think "chia plots check -n"

For us pathetic GUI users there doesn't seem to be a way to properly verify the plots.

If you want to check which plots are still "ok", because they were done with your keys, just start up the guy and let it run, once its synced it should be showing you plots that are nor "farming" and when you hover over it with you mouse it will tell you that the keys are not matching.

**Reply by LachenderTeufel** [reply]
So the plots showing in the GUI are the plots working (has the correct key) am I right?

If so I just can have a look and cross check the available plots from every folder and identify so the "wrong plots".

**Reply by SackofFluor** [reply]
Unabhängig davon, dass niemand mehr abends reinkommt usw. würde ich evt. payouts schon direkt auf eine wirkliche, "eigene" Wallet übertragen. Man weiß nie was da so passieren kann :/

**Reply by SackofFluor** [reply]
Yes that should be alright. You can also check the log files, sometimes duplicates are also mentioned there

**Reply by LachenderTeufel** [reply]
Das ist auf jeden fall der Plan :)
Aber dafür müssten erst einmal payouts vorhanden sein...
Im moment bei 400 Plots = 0 XCH :D

Okay, than I know what to do for the time being... Hope this will work afterwards.

**Reply by LachenderTeufel** [reply]
@SackofFluor Gibt es eine Möglichkeit hier eine DM zu schicken? Würde gerne noch eins zwei Fragen stellen und vielleicht ein wenig Gedankenaustausch betreiben wenn möglich :)

**Comment by YahorL** [comment]
According to statistic, your reward is proportional to your space in the network, so if you have 50% space, you will have 50% reward no matter to what wallets plots are synchronized. In theory, you won't have less reward, it will be distributed between your wallets. Also here https://github.com/Chia-Network/chia-blockchain/discussions/3964 you can find how to check your plots if they relate to your wallet (at the end of the discussion there is a video)

**Reply by LachenderTeufel** [reply]
Okay, understood.
But in the big picture its still not the best way. I  would like to have all running under one wallet. So I need to replace the wrong plots.

So I will have a look on the GUI with the Pool-Key and find out which on is the wrong one...

If this dont work I will have a look with the powershell on each plot

**Reply by YahorL** [reply]
Yeah, you should replot all your plots to make it working for one wallet and you can check each "fingerprint" of the plot in powershell

**Reply by YahorL** [reply]
Yeah, you should replot all your plots to make it working for one wallet and you can check each "fingerprint" of the plot in powershell
